pkg/member/member_control.go

```diff
@@ -116,7 +116,7 @@ func NewMemberControl(etcdConnConfig *brtypes.EtcdConnectionConfig) Control {
 // AddMemberAsLearner add a member as a learner to the etcd cluster
 func (m *memberControl) AddMemberAsLearner(ctx context.Context) error {
 	//Add member as learner to cluster
-	memberURL, err := miscellaneous.GetMemberPeerURL(m.configFile, m.podName)
+	memberPeerURLs, err := miscellaneous.GetInitialAdvertisePeerURLs(m.configFile)
```

Comment:
why you have renamed this function ? I found `GetMemberPeerURL` name more intuitive .

Reply:
Just wanted to be consistent with the actual name. But no strong opinion, I can change this.